firefox-ios/firefox-ios-tests/Tests/ClientTests/Library/Bookmarks/DefaultFolderHeirarchyFetcherTests.swift

```diff
@@ -14,6 +14,7 @@ final class DefaultFolderHeirarchyFetcherTests: XCTestCase {
     override func setUp() async throws {
```

Comment:
nit; Name of class and file `DefaultFolderHeirarchyFetcherTests` -> `DefaultFolderHierarchyFetcherTests`